modules/kubernetes-addons/karpenter/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "helm_config" {
```

Comment:
should we provide functionality to allow customers to specify a provisioner?

Reply:
See the reply for Apporva's comment. Let's discuss this later. Raising a ticket on board